features_and_tools/cql.rst

```diff
@@ -0,0 +1,400 @@
+.. _cql:
+
+CQL Plugin
+============
+
+Introduction
+------------
+
+Firely Server CQL Plugin is a means to use the Firely Server to run the CQL engine. 
+
+This document demonstrates :
+    * Creation of a simple CQL file
+    * Transformation of the CQL file to a library resource using the CQL Packager Tool
+    * Setting up Firely Server CQL Plugin
+    * Getting the result of your CQL measure on a resource using Postman tool
+
+
+Create a CQL File
+-----------------
+
+To start working with Firely Server CQL Plugin, you would need a CQL File.
```

Comment:
Maybe good to refer to documentation were people can find more information on creating a CQL file. Also good to mention the criteria for such a file here.

Reply:
The idea was that customers come up with their own CQL or use the example we provided. I can attach https://cql.hl7.org/03-developersguide.html or https://cql.hl7.org/examples.html. @cheng13231 Do you have a better document?